scriptworker/cot/verify.py

```diff
@@ -803,13 +803,13 @@ def get_all_artifacts_per_task_id(chain, upstream_artifacts):
                 dict_=all_artifacts_per_task_id, key=link.task_id, item='public/task-graph.json'
             )
         # Download actions.json for decision+action task cot verification
-        if link.task_type in DECISION_TASK_TYPES:
-            add_enumerable_item_to_dict(
-                dict_=all_artifacts_per_task_id, key=link.task_id, item='public/actions.json'
-            )
-            add_enumerable_item_to_dict(
-                dict_=all_artifacts_per_task_id, key=link.task_id, item='public/parameters.yml'
-            )
+        # if link.task_type in DECISION_TASK_TYPES:
```

Comment:
For now, we could either create a size 0 file in the Focus decision task, or have a map of `cot_product` to `decision_task_artifacts`. When taskgraph is a generic module, we should get these files for free.

Reply:
Great. Thanks for the idea!